storey/flow.py

```diff
@@ -123,6 +123,26 @@ async def _do(self, event):
             await self._default._do(event)
 
 
+class Recover(Flow):
+    def __init__(self, exception_to_downstream, **kwargs):
+        Flow.__init__(self, **kwargs)
+
+        self._exception_to_downstream = exception_to_downstream
+
+    async def _do(self, event):
+        if not self._outlets or event is _termination_obj:
+            return await super()._do_downstream(event)
+        else:
+            try:
+                await super()._do_downstream(event)
```

Comment:
aren't you supposed to return here as well?

Reply:
Hm, not really? `_do` only returns a termination result.